### DISCUSSION TITLE
Is Laravel 8.x compatible with php 8.2?

### DISCUSSION THREAD

**Discussion by parallels999** [laravel/framework#45439] [Q&A] [answered]
Is Laravel 8.x compatible with php 8.2?

**Comment by henzeb** [comment]
According to this scheme on this page, no. But it is time to upgrade anyway... 8 is EOL 24th of January anyway... 

https://laravel.com/docs/9.x/releases

**Comment by PaolaRuby** [comment, marked as answer]
It would be nice but not
